go.mod

```diff
@@ -18,8 +17,12 @@ require (
 )
 
 require (
+	github.com/ONSdigital/dp-api-clients-go v1.43.0 // indirect
 	github.com/ONSdigital/dp-api-clients-go/v2 v2.254.1 // indirect
 	github.com/ONSdigital/dp-mongodb-in-memory v1.7.0 // indirect
+	github.com/ONSdigital/dp-net v1.5.0
```

Comment:
Something strange has happened here. This dependency (as well as `github.com/justinas/alice v1.2.0` below) should be in the `require` block above, since they're direct dependencies. @willcAND and @jakeandandrews may be able to assist with this.

Reply:
@melsner11Ons what was the resolution for this? Did you run `go mod tidy`

Reply:
Not sure how this was fixed, but after the fix I've also upgraded the rest of the dependencies and I've also run `go mod tidy` as you mentioned.